app/views/admin/dashboards/dashboard.html.erb

```diff
@@ -9,4 +9,11 @@
     Name: <%= customer.first_name %>, <%= customer.last_name %><br>
     Transactions made: <%= customer.transaction_counter %>
   </p>
-<% end %>
\ No newline at end of file
+<% end %>
+
+<section id = "Incomplete_Invoices">
```

Comment:
I did some reading, and it looks like the underscore here needs to be a hyphen to follow convention. so rather than "Incomplete_Invoices", conventionally this section id would be "incomplete-invoices".